From c9b8ea553bc267060474c857e0afae621b6bf65b Mon Sep 17 00:00:00 2001
From: Erez Zadok <ezk@cs.sunysb.edu>
Date: Sun, 25 Nov 2007 18:27:27 -0500
Subject: [PATCH] Unionfs: remove useless debugging messages

These are considered normal behaviour, they don't really reveal any insight
to the person debugging the code, and they tend to clutter console messages.

Signed-off-by: Erez Zadok <ezk@cs.sunysb.edu>
---
 fs/unionfs/commonfops.c |    5 -----
 fs/unionfs/dentry.c     |   15 +++------------
 2 files changed, 3 insertions(+), 17 deletions(-)

diff --git a/fs/unionfs/commonfops.c b/fs/unionfs/commonfops.c
index b33917e..f714e2f 100644
--- a/fs/unionfs/commonfops.c
+++ b/fs/unionfs/commonfops.c
@@ -648,11 +648,6 @@ int unionfs_file_release(struct inode *inode, struct file *file)
 
 	if (fileinfo->rdstate) {
 		fileinfo->rdstate->access = jiffies;
-		pr_debug("unionfs: saving rdstate with cookie "
-			 "%u [%d.%lld]\n",
-			 fileinfo->rdstate->cookie,
-			 fileinfo->rdstate->bindex,
-			 (long long)fileinfo->rdstate->dirpos);
 		spin_lock(&inodeinfo->rdlock);
 		inodeinfo->rdcount++;
 		list_add_tail(&fileinfo->rdstate->cache,
diff --git a/fs/unionfs/dentry.c b/fs/unionfs/dentry.c
index 47a05e5..daedfe2 100644
--- a/fs/unionfs/dentry.c
+++ b/fs/unionfs/dentry.c
@@ -45,12 +45,8 @@ static bool __unionfs_d_revalidate_one(struct dentry *dentry,
 	verify_locked(dentry);
 
 	/* if the dentry is unhashed, do NOT revalidate */
-	if (d_deleted(dentry)) {
-		pr_debug("unionfs: unhashed dentry being "
-			 "revalidated: %*s\n",
-			 dentry->d_name.len, dentry->d_name.name);
+	if (d_deleted(dentry))
 		goto out;
-	}
 
 	BUG_ON(dbstart(dentry) == -1);
 	if (dentry->d_inode)
@@ -451,13 +447,8 @@ static void unionfs_d_release(struct dentry *dentry)
 		printk(KERN_ERR "unionfs: dentry without private data: %.*s\n",
 		       dentry->d_name.len, dentry->d_name.name);
 		goto out;
-	} else if (dbstart(dentry) < 0) {
-		/* this is due to a failed lookup */
-		pr_debug("unionfs: dentry without lower "
-			 "dentries: %.*s\n",
-			 dentry->d_name.len, dentry->d_name.name);
-		goto out_free;
-	}
+	} else if (dbstart(dentry) < 0)
+		goto out_free;  /* due to a (normal) failed lookup */
 
 	/* Release all the lower dentries */
 	bstart = dbstart(dentry);
-- 
1.5.4.3

